src/index.js

```diff
@@ -3,37 +3,18 @@ import ReactDOM from "react-dom";
 import "./index.css";
 import * as serviceWorker from "./serviceWorker";
 import "bootstrap/dist/css/bootstrap.min.css";
-import DocumentManager from "./components/DocumentManager/DocumentManager";
-
-if (!containsCollectionParam(document.location.href)) {
-  insertParam("collection", "d7a2add9-14bf-480e-9b97-96685a006431");
-}
+import DocumentManager from "./components/DocumentManager";
+import { getCollectionFromUrl } from "./util/url";
+import ErrorBoundary from "./components/ErrorBoundary";
 
 ReactDOM.render(
-  <DocumentManager
-    collectionId={getCollectionFromUrl(document.location.href)}
-  />,
+  <ErrorBoundary>
+    <DocumentManager collectionId={getCollectionFromUrl()} />
+  </ErrorBoundary>,
   document.getElementById("root")
 );
 
 // If you want your app to work offline and load faster, you can change
 // unregister() to register() below. Note this comes with some pitfalls.
 // Learn more about service workers: https://bit.ly/CRA-PWA
 serviceWorker.unregister();
-
-function insertParam(key, value) {
```

Comment:
I moved all this logic to `/util` to keep index.js simple and scannable.

Reply:
See above